exercises/practice/list-ops/.meta/reference/list_ops.bal

```diff
@@ -0,0 +1,90 @@
+type IntAccumulator function (int acc, int el) returns int;
+type IntFilter function (int num) returns boolean;
+type IntMapper function (int num) returns int;
+
+public function append(int[] list1, int[] list2) returns int[] {
+    int[][] lists = [list1, list2];
+    return <int[]>concatenate(lists);
+}
+
+public function concatenate(int[][]|int[][][] lists) returns int[]|int[][] {
+    if lists is int[][] {
+        int[] results = [];
+        foreach var list in lists {
+            foreach var elt in list {
+                results.push(elt);
+            }
+        }
```

Comment:
we can use actual type instead of `var`.

```suggestion
        foreach int[] list in lists {
            foreach int elt in list {
                results.push(elt);
            }
        }
```